tests/backup/backup_helper.go

```diff
@@ -2026,3 +2029,46 @@ func MapToKeyValueString(m map[string]string) string {
 	}
 	return strings.Join(pairs, ",")
 }
+
+func GetNextScheduleBackupName(scheduleName string, scheduleInterval time.Duration, ctx context.Context) (string, error) {
```

Comment:
Change the method name to GetNextPeriodicScheduleBackupName, in case of daily/weekly/monthly this method not going to work

Reply:
`CreateIntervalSchedulePolicy` This method takes input as minutes for this we are using `GetNextPeriodicScheduleBackupName`